mon-pix/app/controllers/assessments/challenge.js

```diff
@@ -1,17 +1,23 @@
-import classic from 'ember-classic-decorator';
-import { computed } from '@ember/object';
+import _ from 'lodash';
 import Controller from '@ember/controller';
+import { tracked } from '@glimmer/tracking';
+import progressInAssessment from 'mon-pix/utils/progress-in-assessment';
 
-const { and } = computed;
-
-@classic
 export default class ChallengeController extends Controller {
   queryParams = ['newLevel', 'competenceLeveled'];
-  newLevel = null;
-  competenceLeveled = null;
+  @tracked newLevel = null;
+  @tracked competenceLeveled = null;
+
+  get showLevelup() {
+    return this.model.assessment.showLevelup && this.newLevel;
+  }
 
-  @and('model.assessment.showLevelup', 'newLevel')
-  showLevelup;
+  get pageTitle() {
+    const challengeText = 'Épreuve';
+    const outOfText = 'sur';
+    const stepNumber = progressInAssessment.getCurrentStepNumber(this.model.assessment, _.get(this.model, 'answer.id'));
```

Comment:
@MelanieMEB Pour l'avenir : il me semble que `_.get` est superflu, `this.model.get('answer.id')` devrait marcher non ?